src/Package.php

```diff
@@ -218,6 +220,25 @@ public function hasRoutes(...$routeFileNames): static
         return $this;
     }
 
+    public function hasOptimization(?string $optimize = null, ?string $clear = null, ?string $key = null): static
+    {
+        $this->optimizeCommands[] = [
+            'optimize' => $optimize,
```

Comment:
This will be a command right? Rename the variable to `optimizeCommand`, same for `clearCommand` and `packagenameKey`